packages/geofire/src/index.ts

```diff
@@ -1,4 +1,4 @@
 export { GeoCallbackRegistration } from './GeoCallbackRegistration';
 export { GeoFire } from './GeoFire';
 export { GeoQuery } from './GeoQuery';
-export { GeoFireTypes } from './GeoFireTypes';
\ No newline at end of file
+export * from './GeoFireTypes';
```

Comment:
Is there a reason for this change?  Or was this part of the rollup phase?

Reply:
The [GeoFireTypes.ts](https://github.com/firebase/geofire-js/pull/203/files#diff-0a9e4aa8acec255b72e32ce315d46310ac157cb6f842bbf1600859abae01b5d5) is new, meant to isolate types that `geofire-utils` needs from types that only `geofire` proper need. In this last commit I removed the unnecessary namespacing after a chat with @jamesdaniels on this antipattern.